Sources/Common/Extensions/URLExtension.swift

```diff
@@ -138,6 +138,13 @@ extension URL {
     public var isValid: Bool {
         guard let navigationalScheme else { return false }
 
+        // Checks if is email if so is not considered a valid url
+        var trimmedURL = absoluteString.dropping(prefix: "http://")
```

Comment:
Not a big deal, but you can use `NavigationalScheme.http.rawValue` and also `NavigationalScheme.separator`